### ISSUE TITLE
Cannot set multiple Authorized User Groups with CLI

### ISSUE THREAD

**Issue by littlevale** [veyon/veyon#537]
I've been trying to add a second group to the authorized users using the CLI. I haven't fathomed the syntax.

If I set the access group using the configurator to, say, Administrators and `HomeUsers` I can then read the configuration with

`veyon-cli config get AccessControl/AuthourizedUserGroups`

which returns

`("Administrators","HomeUsers")`

If I then set the authorized user groups to just Administrators using the configurator, restart the Veyon service, then use

`veyon-cli config set AccessControl/AuthourizedUserGroups ("Administrators","HomeUsers")`

the CLI return `[OK]` but the list of authorized user groups in the configurator is now empty.

I've tried other variations, like `Administrators,HomeUsers` or `Administrators HomeUsers`. The CLI returns `[OK]` but in each case I get either just the first named group, or a blank list.

I can set just a single group with the CLI - how can I set two or more groups?

**Comment by tobydox** [comment]
Thanks this issue! The `config` plugin did not support setting string list properties such as `AccessControl/AuthorizedUserGroups` however I just fixed this shortcoming. Starting with the upcoming Veyon 4.2.3 release you'll be able to set groups via

`veyon-cli config set AccessControl/AuthorizedUserGroups "Administrator;HomeUsers;My Group"`

If it doesn't work, make sure to `unset` the property before in case the data type is messed up internally.

**Comment by littlevale** [comment]
I've downloaded Veyon 4.2.3 and tried this again but with no success. I used `veyon-cli config clear` to clear the configuration, then `veyon-cli service restart` just to be sure. Checking the configurator panel now lists no user groups, `Access Restricted To User Groups` is unchecked, as is `DomainGroupsEnabled`. `veyon-cli config list` agrees with this. So far, so good.

Now, I tried setting the two flags with 
```
veyon-cli config set AccessControl/AccessRestrictedToUserGroups true
veyon-cli config set AccessControl/DomainGroupsEnabled true
veyon-cli service restart
```
Checking with the configurator shows the new settings have been applied.

Lastly
```
veyon-cli config set AccessControl/AuthorizedUserGroups "Administrators;WKSAdmin"
veyon-cli service restart
```
and check again with the configurator: the authorised user list is blank.

Listing the config gives
`AccessControl/AuthorizedUserGroups=Administrators;WKSAdmin`

Adding the required user groups with the configurator and listing again now gives
`AccessControl/AuthorizedUserGroups=Administrators;WKSAdmin;WKSAdmin;Administrators`

Exporting the configuration to a JSON file now gives:
```
"AccessControl": {
  "AccessRestrictedToUserGroups": "true",
  "AuthorizedUserGroups:: [
    "Administrators;WKSAdmin",
    "Administrators",
    "WKSAdmin"
  ],
  "DomainGroupsEnabled": "true"
},
```
Clearly, the CLI utility is still not parsing the string properties properly.
